src/test/java/uk/gov/hmcts/ethos/replacement/docmosis/service/NocNotificationServiceTest.java

```diff
@@ -113,8 +113,6 @@ void sendNotificationsShouldSendThreeNotifications() {
         nocNotificationService.sendNotificationOfChangeEmails(callbackRequest, caseData);
         // Claimant
         verify(emailService, times(1)).sendEmail(any(), eq("claimant@represented.com"), any());
-        // Previous respondent
-        verify(emailService, times(1)).sendEmail(any(), eq("oldres@rep.com"), any());
```

Comment:
Why remove this line? Could we set it to times(0) if no longer required

Reply:
changed to use times(0) as discussed